src/core_ocean/mode_init/mpas_ocn_init_lock_exchange.F

```diff
@@ -245,13 +307,6 @@ subroutine ocn_init_setup_lock_exchange(domain, iErr)!{{{
                activeTracers(index_salinity, :, iCell) = config_lock_exchange_salinity
             end if
 
-            ! Set debugging tracer
-            if ( associated(debugTracers) ) then
-               do k = 1, nVertLevels
-                  debugTracers(index_tracer1, k, iCell) = 1.0_RKIND
-               enddo
-            end if
-
```

Comment:
I'm not an expert in debug tracers but my sense was that `debugTracers(1, :, :)` was purposefully set to all ones to be a check on conservation in the model.  It might be better to modify `debugTracer(2 :, :)` to serve the purpose you have above.

Reply:
Thanks, I will put back the `debugTracer(index_tracer1, :, :)` content and use `index_tracer2` for my test case.

Reply:
Sounds good.  Make sure that actually works.

Reply:
I updated the code and the test case (https://github.com/MPAS-Dev/compass/pull/87) accordingly. I re-ran the case and plot the outputs with no issues.